src/pages/index.js

```diff
@@ -1,37 +1,173 @@
+});
+
+//Form Validator Class
+const editProfileFormValidator = new FormValidator(
+  config,
+  editProfilePopup.popupForm
+);
+
+const addPlaceFormValidator = new FormValidator(
+  config,
+  addPlacePopup.popupForm
+);
+
+const profilePicFormValidator = new FormValidator(
+  config,
+  profilePicPopup.popupForm
+);
+
+//Open Buttons
+
+const editProfileMOB = document.querySelector(
+  "#edit-profile-modal-open-button"
+);
 const addCardMOB = document.querySelector("#add-card-modal-open-button");
+const profilePic = document.querySelector("#profile-photo");
+
+//Form Placeholders
 const userNameInput = document.querySelector("#edit-profile-name-placeholder");
-const userJobInput = document.querySelector("#edit-profile-subtitle-placeholder");
+const userJobInput = document.querySelector(
+  "#edit-profile-subtitle-placeholder"
+);
+
+//User Info
+const profileName = document.querySelector("#profile-name");
+const profileAbout = document.querySelector("#profile-subtitle");
+
+//Modals
+const editProfilePicModal = document.querySelector("#edit-pic-modal");
+const addCardModal = document.querySelector("#add-card-modal");
+const editProfileModal = document.querySelector("#edit-profile-modal");
 
+//Popups
+const popups = [
+  maxImagePopup,
+  editProfilePopup,
+  addPlacePopup,
+  profilePicPopup,
+  confirmDeletePopup,
+];
+
+//Form Validators
+const formValidators = [
+  editProfileFormValidator,
+  addPlaceFormValidator,
+  profilePicFormValidator,
+];
+
+//Card Container
+let cardSection;
 
 //***************************************************************************************************************************//
 //                                                 Functions                                                                 //
 //***************************************************************************************************************************//
 
-const createCard = (cardData, method = "append") => {
-  const cardTemplate = new Card(cardData, "#card-template", handleImageClick);
+//Creates new Cards
+function addNewCard(cardData, method = "append") {
+  const cardTemplate = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleConfirmDelete,
+    handleLikeCard
+  );
   const cardElement = cardTemplate.getCardElement();
+  cardTemplate.renderLikes();
 
   cardSection.addItem(cardElement, method);
-};
+}
+
+//Sets event Listeners
+function setEventListeners(popups) {
+  popups.forEach((popup) => {
+    popup.setEventListeners();
+  });
+}
+
+//Sets Form validation
+function setFormValidation(formValidators) {
+  formValidators.forEach((formValidator) => {
+    formValidator.enableValidation();
+    formValidator.toggleButtonState();
+  });
+}
+
+//Fetches then sets User Info
+function setUserInfo() {
+  api
+    .getUserInfo()
+    .then((userInfo) => {
+      profileName.textContent = userInfo.name;
+      profileAbout.textContent = userInfo.about;
+      profilePic.src = userInfo.avatar;
```

Comment:
Class `UserInfo` should set the entire user’s data (including the **avatar**)

You should use `userInfo.setUserInfo` here


```js

    .then((userData) => {
      userInfo.setUserInfo(userData)
    })
```